addons/l10n_cz/data/template/account.tax-cz.csv

```diff
@@ -3,61 +3,61 @@
 "","","","","","","","","","","","100","tax","invoice","1","","","","","",""
 "","","","","","","","","","","","100","base","refund","1","","VAT 26","","","",""
 "","","","","","","","","","","","100","tax","refund","1","","","","","",""
-"l10n_cz_12_purchase_goods_eu","12% EU G","12% - EU Purchase of Goods","12% VAT","consu","purchase","12","percent","tax_group_vat_12","fiscal_position_eu_private_cz,fiscal_position_domestic_cz","","100","base","invoice","1","","VAT 4 Base","12% EU G","12% - Pořízení zboží z jiného státu EU","12% DPH",""
+"l10n_cz_12_purchase_goods_eu","12% EU G","12% - EU Purchase of Goods","12% VAT","consu","purchase","12","percent","tax_group_vat_12","fiscal_position_intra_community","","100","base","invoice","1","","VAT 4 Base","12% EU G","12% - Pořízení zboží z jiného státu EU","12% DPH",""
 "","","","","","","","","","","","100","tax","invoice","1","chart_cz_343112","VAT 4 Tax","","","",""
 "","","","","","","","","","","","-100","tax","invoice","-1","chart_cz_343212","","chart_cz_343212","","",""
 "","","","","","","","","","","","100","base","refund","1","","VAT 4 Base","","","",""
 "","","","","","","","","","","","100","tax","refund","1","chart_cz_343112","VAT 4 Tax","","","",""
 "","","","","","","","","","","","-100","tax","refund","-1","","","","","",""
-"l10n_cz_supply_goods_eu","0% EU G","EU Sale of Goods","The buyer is obligated to fill in the VAT amounts and pay the tax.","consu","sale","0","percent","tax_group_vat_0","","","100","base","invoice","1","","VAT 20","0% EU G","Dodání zboží do jiného státu EU","Daň odvede zákazník","Exempt intra-community supplies - Article 138 of the Directive 2006/112/EC"
+"l10n_cz_supply_goods_eu","0% EU G","0% Intra-Community (Goods)","The buyer is obligated to fill in the VAT amounts and pay the tax.","consu","sale","0","percent","tax_group_vat_0","fiscal_position_intra_community","l10n_cz_21_domestic_supplies,l10n_cz_12_domestic_supplies,l10n_cz_export_goods,l10n_cz_other_transaction_deduction","100","base","invoice","1","","VAT 20","0% EU G","0 % uvnitř Společenství (zboží)","Daň odvede zákazník","Exempt intra-community supplies - Article 138 of the Directive 2006/112/EC"
```

Comment:
I think we tend not to put the export here.
As normally, it's the export that overrides this tax.
I've no idea if it's a big deal

Reply:
@bewir-odoo any suggestion on this ?